src/java/com/unifina/feed/StreamrMessage.java

```diff
@@ -3,17 +3,33 @@
 import java.util.Date;
 import java.util.Map;
 
+import com.unifina.data.StreamrBinaryMessage;
 import com.unifina.feed.map.MapMessage;
 
 public class StreamrMessage extends MapMessage {
 
 	private final int partition;
 	private final String streamId;
+	private final byte signatureType;
```

Comment:
`StreamrMessage` should not know anything about the byte representation (or `StreamrBinaryMessage`) - perhaps we should make this an enum instead?